backend.c

```diff
@@ -269,7 +269,8 @@ static bool fio_io_sync(struct thread_data *td, struct fio_file *f)
 
 	io_u->ddir = DDIR_SYNC;
 	io_u->file = f;
-	io_u_set(td, io_u, IO_U_F_NO_FILE_PUT);
```

Comment:
Will there be a problem here with reference counts if the job is using multiple files and the `io_u` that is in the requeue list refers to a file that is different from the one that is the target of the sync operation?